private/sass/layout/_iconography.scss

```diff
@@ -24,1633 +23,1633 @@
 // INFO: use unicode characters for accessibility reasons and use aria-hidden="true" for decorative icons
 // DOCS: http://filamentgroup.com/lab/bulletproof_icon_fonts.html
 .fa-glass:before {
-  content: "\f000";
+    content: "\f000";
```

Comment:
General comment: when you include many formatting changes in your PR, mixed with more substantial changes, you're making it harder to review and more likely that the reviewer will miss important bits. If it all possible, it's best to do the reformatting in a separate branch.